src/pages/resultsView/coExpression/CoExpressionTab.tsx

```diff
@@ -138,7 +122,13 @@ export default class CoExpressionTab extends React.Component<ICoExpressionTabPro
 
     @bind
     private onSelectGeneticEntity(geneticEntityId:string) {
-        this._selectedGeneticEntityId = geneticEntityId;
+        if (this.props.store.geneticEntities.isComplete) {
```

Comment:
you might consider throwing an error here because the UI should never allow a user to selected a geneticEntity before geneticEntites are complete.  but not critical

Reply:
@alisman Done

Reply:
its better to throw an error object so that the stack trace is available which i believe its not if you just throw a string

Reply:
its better to throw an error object so that the stack trace is available which i believe its not if you just throw a string